src/index.html

```diff
@@ -16,7 +16,16 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="stopwatch">
+    <section class="stopwatch__object">
+      <article class="stopwatch__elements"></article>
+      <div
+        class="stopwatch__elements stopwatch__elements--second-hand stopwatch__elements--second-hand--speed-up"
```

Comment:
BEM naming convention error: Modifiers should not be used alone. Instead of `stopwatch__elements--second-hand--speed-up`, you should have `stopwatch__elements stopwatch__elements--second-hand stopwatch__elements--speed-up` if 'speed-up' is a modifier that can apply to different elements.